oeps/oep-0010.rst

```diff
@@ -103,10 +102,29 @@ parts of providing a finished quality product, and we don't want to overlook
 them.
 
 
+Dependencies
+------------
+
+Our software is built atop other software layers supported by their creators.
+It's important to consider the support windows for those layers when choosing
+which version to use.  An Open edX release is supported by edX until the next
+release, so for about six months.  The supporting layers must be supported by
+their developers for the entire Open edX release.
+
+Typically this means choosing Long Term Support (LTS) versions of the
+supporting layers, but it's possible shorter-term support versions will provide
+the support needed.
+
+The layers in question here are Django, Python, and Ubuntu.  Here's a `calendar
+of the known support windows`__ and how they overlap with Open edX plans.
+
+.. __: https://docs.google.com/spreadsheets/d/11DheEtMDGrbA9hsUvZ2SEd4Cc8CaC4mAfoV8SVaLBGI
```

Comment:
For now, this will work, but I'd recommend putting a note in the spreadsheet to make it clear that it's linked from this OEP, just so you/others don't make changes there in the future that negatively impact the message here.